src/schemas/schemas.ts

```diff
@@ -0,0 +1,29 @@
+export const getLikedItems = {
+  params: { 
+    memberId: { $ref: 'http://graasp.org/#/definitions/uuid' },
+  },
+  response: {
+    200: {
+      type: 'array',
+    },
+  },
+};
+
+export const getLikeCount = {
+  params: { 
+    itemId: { $ref: 'http://graasp.org/#/definitions/uuid' },
+  },
+};
+
+export const create = {
+  params: { 
+    itemId: { $ref: 'http://graasp.org/#/definitions/uuid' },
+    memberId: { $ref: 'http://graasp.org/#/definitions/uuid' },
+  }
+};
+
+export const deleteOne = {
+  params: {
+    entryId: { $ref: 'http://graasp.org/#/definitions/uuid' },
+  },
+};
```

Comment:
Actually could you also precise the return value types? So we have control on the value we return.